hoomd/DomainDecomposition.cc

```diff
@@ -371,9 +375,18 @@ bool DomainDecomposition::findDecomposition(unsigned int nranks,
                     continue;
                 if (nx_try * ny_try * nz_try != nranks)
                     continue;
-                double surface_area = L.x * L.y * (double)(nz_try - 1)
-                                      + L.x * L.z * (double)(ny_try - 1)
-                                      + L.y * L.z * (double)(nx_try - 1);
+                if (is2D && nz_try > 1)
+                    continue;
+                double surface_area;
+                if (is2D)
+                    surface_area = L.x * (ny_try - 1) + L.y * (nx_try - 1);
```

Comment:
```suggestion
                    {
                    surface_area = L.x * (ny_try - 1) + L.y * (nx_try - 1);
                    }
```
`clang-format` doesn't catch this one, but we should use curly braces even on small if blocks.